windows/jitsi-meet/render.js

```diff
@@ -0,0 +1,167 @@
+const server = require('../../modules/httpcontrol').HttpServer;
+const listeningPort = 1025;
+const targetPort = 1024;
+
+/**
+ * The jitsi-meet api.
+ */
+let api;
+
+window.onload = function() {
+    server.init(listeningPort);
+    server.onReceivedCommand((type, args) => {
+        handleCommand(type, args);
+    });
+
+    document.onkeydown = _onKeyDown;
+    document.getElementById('enter_room_button').onclick = function() {
+        const roomName = document.getElementById('enter_room_field').value;
+        const command = {
+            type: 'join',
+            args: roomName
+        };
+
+        triggerLoader();
+        sendHttpCommand(`http://localhost:${targetPort}`, command,
+            data => {
+                console.log(data);
+            }
+        );
+    };
+
+    /**
+     * Listener for Eneter key down event.
+     *
+     * @param {event} event - key down event
+     * @returns {null}
+     */
+    function _onKeyDown(event) {
+        if (event.keyCode === /* Enter */ 13) {
+            const roomName = document.getElementById('enter_room_field').value;
+            const command = {
+                type: 'join',
+                args: roomName
+            };
+
+            triggerLoader();
+            sendHttpCommand(`http://localhost:${targetPort}`, command,
+                data => {
+                    console.log(data);
+                }
+            );
+        }
+    }
+};
+
+/**
+ * Loads Jitsi-meet page in iframe with given room name.
+ *
+ * @returns {null}
+ */
+function triggerLoader() {
+    document.onkeydown = null;
+    document.getElementById('welcome_page_main').classList.add('animated');
+    document.getElementById('welcome_page_main').classList.add('fadeOutUpBig');
+    setTimeout(() => {
+        document.getElementById('cs-loader-inner')
+            .classList.add('cs-loader-inner');
+    }, 1000);
+}
+
+/**
+ * Cteates the iframe that will load Jitsi Meet.
+ *
+ * @param {string} room - conference room name
+ * @returns {null}
+ */
+function createJitsiIframe(room) {
+    const domain = 'meet.jit.si';
```

Comment:
let's take this from some config.js module or an env variable. Also when we require the script.